userguide/storage.rst

```diff
@@ -1337,6 +1352,183 @@ should not be used with :ref:`Windows (SMB) Shares`.
+   |                   |                  | :ref:`directory service <Directory Services>` appear in the drop-down menu.                                |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | Group             | drop-down menu   | Select the group to control the dataset. This group always has permissions to read or write the ACL and    |
+   |                   |                  | read or write attributes. Groups manually created or imported from a                                       |
+   |                   |                  | :ref:`directory service <Directory Services>` appear in the drop-down menu.                                |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | Tag               | drop-down menu   | Access Control Entry (ACE) type. Select *User* or *Group* to define a specific user or group for this      |
+   |                   |                  | entry, *owner@* to apply this entry to the selected :guilabel:`User`, *group@* to apply this entry to the  |
+   |                   |                  | selected :guilabel:`Group`, or *everyone@* to apply this entry to all users and groups. See                |
+   |                   |                  | `setfacl(1) NFSv4 ACL ENTRIES <https://www.freebsd.org/cgi/man.cgi?query=setfacl>`__.                      |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | User              | drop-down menu   | User account to which this ACL entry applies. Only visible when *User* is the chosen :guilabel:`Tag`.      |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | Group             | drop-down menu   | Group to which this ACL entry applies. Only visible when *Group* is the chosen :guilabel:`Tag`.            |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | ACL Type          | drop-down menu   | How the :guilabel:`Permissions` are applied to the tagged account. Choose *Allow* to grant the specified   |
+   |                   |                  | permissions and *Deny* to restrict the specified permissions.                                              |
+   +-------------------+------------------+------------------------------------------------------------------------------------------------------------+
+   | Permissions Type  | drop-down menu   | Define which set of :guilabel:`Permissions` to display. *Basic* shows general, broadly applied permissions |
```

Comment:
It's not defining a set, it's just selecting.  Also, "broadly applied" is not very clear.  Also, break the sentence here.
```suggestion
   | Permissions Type  | drop-down menu   | Select which set of :guilabel:`Permissions` to display. *Basic* shows general permissions.                 |
```